src/main/java/hudson/plugins/sonar/client/SQProjectResolver.java

```diff
@@ -52,7 +52,13 @@ public ProjectInformation resolve(String projectUrl, String ceTaskId, String ins
     try {
       String projectKey = extractProjectKey(projectUrl);
       String serverUrl = extractServerUrl(projectUrl);
-      WsClient wsClient = new WsClient(client, serverUrl, inst.getSonarLogin(), inst.getSonarPassword());
+      
+      WsClient wsClient;
+      if(StringUtils.isNotEmpty(inst.getServerAuthenticationToken())) {
+        wsClient = new WsClient(client, serverUrl, inst.getServerAuthenticationToken(), null);
+      } else {
+       wsClient = new WsClient(client, serverUrl, inst.getSonarLogin(), inst.getSonarPassword());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)